Sources/PrinceOfVersions/ResponseModels/AppStoreUpdateData/AppStoreUpdateInfo.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  AppStoreUpdateInfo.swift
+//  PrinceOfVersions
+//
+//  Created by Jasmin Abou Aldan on 06/02/2020.
+//  Copyright © 2020 Infinum Ltd. All rights reserved.
+//
+
+import Foundation
+
+#if canImport(UIKit)
+import UIKit
+#elseif canImport(AppKit)
+import AppKit
+#endif
+
+public struct AppStoreUpdateInfo: Codable {
+
+    // MARK: - Internal properties -
+
+    static internal var bundle: Bundle = .main
+
+    internal var notificationFrequency: NotificationType = .always
+
+    internal let resultCount: Int?
```

Comment:
Do you really need a result count when you have a results one line below and it can produce it's own count?

Reply:
It is removed.